### PR TITLE
Feature/faq contact pages rm header info

### PR DESCRIPTION
**Pull request by MichaelVessia**

### CODE REVIEW

**Comment by MichaelVessia** [comment]
Should close #905 and #906

File: server/package-lock.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "lockfileVersion": 1
```
Comment (sstangl) [inline server/package-lock.json]: What's this file? Probably shouldn't be here.

File: server/templates/data.html.tera [hunk 2]
```diff
@@ -7,6 +7,14 @@
   </div>
 {% endblock topbar_hack_end %}
 
+{% block controls %}
+  <div id="controls">
+    <div id="controls-right">
+      {% include "langselect" %}
```
Comment (sstangl) [inline server/templates/data.html.tera]: This will also require the `{% include "langselect-script" %}`.

File: server/templates/header.tera [hunk 3]
```diff
@@ -10,8 +10,4 @@
   <a href="https://instagram.com/openpowerlifting" class="nav__link"><i class="fa fa-instagram"></i></a>
   <a href="https://facebook.com/openpowerlifting" class="nav__link"><i class="fa fa-facebook"></i></a>
   <a href="javascript:void(0)" class="nav__link header-hamburger" onClick="document.getElementById('header').classList.toggle('responsive')">&#9776;</a>
-
-  <span class="siteinfo">
-    {{header.num_entries}} Entries &mdash; {{header.num_meets}} Meets &mdash; Updated Sometime
```
Comment (sstangl) [inline server/templates/header.tera]: Need to also change the `lifter.rs` and `meet`rs to stop passing those things to their `Context` objects.

File: server/templates/data.html.tera (hunk 2 above)
Comment (MichaelVessia) [reply server/templates/data.html.tera]: @sstangl It's in the custom javascript block on the bottom, is that alright? Just basing it on the pre-existing `lifter.html.tera`

File: server/templates/header.tera (hunk 3 above)
Comment (MichaelVessia) [reply server/templates/header.tera]: @sstangl Fairly certain I did this can you confirm?

File: server/templates/data.html.tera (hunk 2 above)
Comment (sstangl) [reply server/templates/data.html.tera]: Yes

File: server/package-lock.json (hunk 1 above)
Comment (MichaelVessia) [reply server/package-lock.json]: No idea! removed

**Pull request merged**